src/ReviewPanel.jsx

```diff
@@ -1,32 +1,29 @@
 import React, {PropTypes} from 'react'
-import { Panel, Row, Col } from 'react-bootstrap'
+import { Panel, Col } from 'react-bootstrap'
 
 const ReviewPanel = ({review}) => {
   if(!review) return <div />;
+
+  const body=review.body;
   const time = review.timestamp;
   const good = review.positive;
   const reviewer = review.team_member_name;
   const header = reviewer + " on " + new Date(time).toDateString();
-  let flags = null;
```

Comment:
Like there used to be a little div here that I had the plus/minus in. I had thought that maybe sometime we would put more widgets there.